src/App.tsx

```diff
@@ -1,18 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+
+import { Pagination } from './components/Pagination';
+
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
+const units = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [items] = useState(units);
+
+  const handleSelect = (e: React.BaseSyntheticEvent) => {
+    setPerPage(parseFloat(e.target.value));
```

Comment:
Instead of using parseFloat, consider using parseInt to avoid potential issues with floating-point numbers 😊